Framework/Geometry/src/Rasterize.cpp

```diff
@@ -122,6 +122,25 @@ calculateHollowCylinder(const V3D &beamDirection,
                                  numAnnuli);
 }
 
+const V3D calculatePos(const double phi, const double R, const double z,
```

Comment:
Maybe calling in `calculatePosInCylinder` would be clearer. I don't know if its necessary since it is only in this file.